.gitignore

```diff
@@ -0,0 +1,3 @@
+
+res/.DS_Store
+.DS_Store
```

Comment:
I'll add a `.gitignore` in just a sec; would you be able to revert these changes so they don't clash with the one I'm going to add? Once I'm done with that you can [rebase your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork) and there should be no issues with `.DS_Store` files.